hologram_dashboard/fetcher/hologram_data_source_availability_fetcher.cc

```diff
@@ -20,6 +20,11 @@ namespace wireless_android_play_analytics{
 
 void HologramDataSourceAvailabilityFetcher::Process() {
     // TODO(alexanderlin): add implementation.
+    assert(FLAGS_config_file_path is not empty)
+    // AcquireConfig(FLAGS_config_file_path)
+    // FetchFromDatabase()
+    // GetStatus(todays date)
+    // SendToDataBase()
```

Comment:
I am still a bit confused about the relationship between each functions. I expect a complete call stack. Define the apis just like the final version but you could leave the implementation empty.

Probably something like this (I might be wrong on the design, but this is a rough idea):
HologramDataAvailability* data_availability = FetchFromDatabase();
UpdateWithLatestStatus(date, data_availability);
UpdateDatabase(data_availability);